examples_utils/benchmarks/README.md

```diff
@@ -0,0 +1,53 @@
+# Applications benchmarking
```

Comment:
Nice readme! As this readme is going to be public it might be worth adding an example or further explanation of what the benchmarks are or used for